ops/pebble.py

```diff
@@ -3211,6 +3317,44 @@ def get_notices(
         resp = self._request('GET', '/v1/notices', query)
         return [Notice.from_dict(info) for info in resp['result']]
 
+    def get_identities(self) -> Dict[str, Identity]:
```

Comment:
Please merge latest `main` (or rebase, whichever) and add instrumentation on this the the other two methods.